docs/broker/filtering.md

```diff
@@ -266,8 +265,8 @@ _The `>` syntax is a standard yaml multiline string._
 spec:
   filter:
     expression: >
-      ce.type == "com.github.pull.create" ||
-      (ce.type == "com.github.issue.create" && ce.source.matches("proposals")
```

Comment:
@mattmoor I think we wanted it formatted like this. Know of a way to tell prettier this is the case?